arwen/hosttest/execution_test.go

```diff
@@ -317,15 +321,23 @@ func TestExecution_MultipleArwens_OverlappingDifferentVMHooks(t *testing.T) {
 	input.GasProvided = 1000000
 	input.Function = get
 
-	host1, instanceRecorder1 := test.DefaultTestArwenForCallWithInstanceRecorderMock(t, code, nil)
+	executorFactory1 := contextmock.NewExecutorRecorderMockFactory()
+	host1 := test.NewTestHostBuilder(t).
+		WithExecutorFactory(executorFactory1).
+		WithBlockchainHook(test.BlockchainHookStubForCall(code, nil)).
+		Host()
 	defer func() {
 		host1.Reset()
 	}()
 	_, _, _, _, runtimeContext1, _, _ := host1.GetContexts()
 	runtimeContextMock := contextmock.NewRuntimeContextWrapper(&runtimeContext1)
 	host1.SetRuntimeContext(runtimeContextMock)
 
-	host2, instanceRecorder2 := test.DefaultTestArwenForCallWithInstanceRecorderMock(t, code, nil)
+	executorFactory2 := contextmock.NewExecutorRecorderMockFactory()
```

Comment:
Consider a cleaner name for this constructor. Is this an executor recorder? A mock factory of executors? Or is it `Mock` simply because it is used in tests?

Reply:
I changed it to `ExecutorDebugger`, since I realized it was just a wrapper.